cavern/src/main/java/org/opencadc/cavern/files/GetAction.java

```diff
@@ -116,7 +121,18 @@ public void doAction() throws Exception {
                 return;
             }
             
-            // TODO: get node so we can put appropriate properties into HTTP headers
+            // set HTTP headers.  To get node, resolve links but no authorization (null authorizer) 
+            NodePersistence nodePersistence = new FileSystemNodePersistence();
+            PathResolver pathResolver = new PathResolver(nodePersistence, true);
+            Node node = pathResolver.resolveWithReadPermissionCheck(nodeURI, null, true);
+            log.info("BM: req: " + nodeURI + " res: " + node.getUri() + " md5: " + node.getPropertyValue(VOS.PROPERTY_URI_CONTENTMD5));
+            String contentEncoding = node.getPropertyValue(VOS.PROPERTY_URI_CONTENTENCODING);
+            String contentLength = node.getPropertyValue(VOS.PROPERTY_URI_CONTENTLENGTH);
+            String contentMD5 = node.getPropertyValue(VOS.PROPERTY_URI_CONTENTMD5);
+            syncOutput.setHeader("Content-Disposition", "inline; filename=" + node.getName());
```

Comment:
You should add Content-Type to the props here... user-set like encoding but should be supported